src/pages/billing/SpendReport/SpendReport.ts

```diff
@@ -48,13 +48,14 @@ interface ProjectCost {
   spend: string;
   compute: string;
   storage: string;
+  workspaceinfrastructure: string;
```

Comment:
Nit: This should be lowerCamelCase, if it isn't cased in order to directly map to a key in a json or something like that.